### ISSUE TITLE
Bug reprot when to define filled members in HydroDyn driver

### ISSUE THREAD

**Issue by chenkongenvision** [OpenFAST/openfast#310]
**Bug description**

When I define a very simple model with filled members in HydroDyn, I found that if there are some unfilled member in the FillMList. Then the Ballsted Volume in the output .HD.sum file is calculated incorrectly. For example: If we define MEMBER JOINTS, MEMBER CROSS-SECTION PROPERTIES, MEMBERS and FILLED MEMBERS as following:
```
---------------------- MEMBER JOINTS -------------------------------------------
             4   NJoints        - Number of joints (-)   [must be exactly 0 or at least 2]
JointID   Jointxi     Jointyi     Jointzi  JointAxID   JointOvrlp   [JointOvrlp= 0: do nothing at joint, 1: eliminate overlaps by calculating super member]
   (-)     (m)         (m)         (m)        (-)       (switch)
    1     0.00000     0.00000   -20.00010      1            0
    2     0.00000     0.00000   -10.00000      1            0
    3     0.00000     0.00000     0.00000      1            0
    4     0.00000     0.00000    10.00000      1            0
---------------------- MEMBER CROSS-SECTION PROPERTIES -------------------------
             2   NPropSets      - Number of member property sets (-)
PropSetID    PropD         PropThck
   (-)        (m)            (m)
    1        6.00000        0.06000
    2        8.00000        0.06000
-------------------- MEMBERS -------------------------------------------------
             3   NMembers       - Number of members (-)
MemberID  MJointID1  MJointID2  MPropSetID1  MPropSetID2  MDivSize   MCoefMod  PropPot   [MCoefMod=1: use simple coeff table, 2: use depth-based coeff table, 3: use member-based coeff table] [ PropPot/=0 if member is modeled with potential-flow theory]
  (-)        (-)        (-)         (-)          (-)        (m)      (switch)   (flag)
    1         1          2           1            1         0.5000       1      FALSE
    2         2          3           1            1         0.5000       1      FALSE
    3         3          4           1            1         0.5000       1      FALSE
---------------------- FILLED MEMBERS ------------------------------------------
             1   NFillGroups     - Number of filled member groups (-) [If FillDens = DEFAULT, then FillDens = WtrDens; FillFSLoc is related to MSL2SWL]
FillNumM FillMList             FillFSLoc     FillDens
(-)      (-)                   (m)           (kg/m^3)
3        1 2 3                 0             DEFAULT
```
Obviously, the member 3 is not filled according to the FillFSLoc value and member definition. Then the HydroDyn return the Ballasted Volume:
```
  Ballasted Volume     :    8.14643E+02
```
which is the filled volume for the whole structure.

 Of course, if we change the parameters to:
```
---------------------- FILLED MEMBERS ------------------------------------------
             1   NFillGroups     - Number of filled member groups (-) [If FillDens = DEFAULT, then FillDens = WtrDens; FillFSLoc is related to MSL2SWL]
FillNumM FillMList             FillFSLoc     FillDens
(-)      (-)                   (m)           (kg/m^3)
2        1 2                 0             DEFAULT
```
or
```
---------------------- FILLED MEMBERS ------------------------------------------
             1   NFillGroups     - Number of filled member groups (-) [If FillDens = DEFAULT, then FillDens = WtrDens; FillFSLoc is related to MSL2SWL]
FillNumM FillMList             FillFSLoc     FillDens
(-)      (-)                   (m)           (kg/m^3)
3        1 2 3                 0.0001             DEFAULT
```
HydroDyn could return the correct Ballasted Volume as expected:
```
  Ballasted Volume     :    5.43096E+02
```
I know this issue can be avoided by defining FILLED MEMBERS without any UNFILLED members. But it will be perfect if HydroDyn could accept UNFILLED members in the FillMList and calcualte Ballasted Volume correctly. At least I would expect some error messages to stop the program if some UNFILLED members are defined in the FillMList. 

**To Reproduce**

1. Compile with 'VS2017'
2. Run the attached case with HydroDyn standalone
[HydroDyn_test_openfast.zip](https://github.com/OpenFAST/openfast/files/3420143/HydroDyn_test_openfast.zip)

3. Open 'OC3Monopile_HydroDyn_flooded.HD.sum' output
4. See ```Ballasted Volume``` value

**Expected behavior**

The correct value should be ```5.43096E+02```

**Screenshots**

**OpenFAST Version**

Please provide as much detail as possible including git commit. The best information is a screenshot of the OpenFAST system description that prints when running OpenFAST:

```
**************************************************************************************************
HydroDyn Driver

Copyright (C)  National Renewable Energy Laboratory
Copyright (C)  Envision Energy USA LTD

This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
See the "LICENSE" file distributed with this software for details.
**************************************************************************************************

 Running HydroDyn Driver
 a part of OpenFAST - v2.1.0-dirty
 linked with NWTC Subroutine Library
```

**System Information (please complete the following information):**
 
 - OS: Windows 7
 - Compiler: Intel Visual Fortran Compiler 19.0
 - Compiler settings: default settings in the (openfast repo)\vs-build\HydroDyn

**Comment by jjonkman** [comment]
Thanks @chenkongenvision.  Definitely sounds like a bug.

I haven't looked closely at this, but there was a bug in the external buoyancy reported in the summary file that was fixed with https://github.com/OpenFAST/openfast/pull/214.  Perhaps this is the same issue, but this time with the internal buoyancy of filled members?

@ghaymanNREL -- Could you look into this?

**Comment by jjonkman** [comment]
I believe this bug was fixed in OpenFAST v2.6 (https://github.com/OpenFAST/openfast/releases/tag/v2.6.0).
